hvplot/converter.py

```diff
@@ -1330,7 +1330,7 @@ def method_wrapper(ds, x, y):
                 opts['rescale_discrete_levels'] = self._plot_opts['rescale_discrete_levels']
         else:
             operation = rasterize
-            eltype = 'Image'
+            eltype = 'ImageStack' if self.by else 'Image'
```

Comment:
We should safeguard this so it only happens with a holoviews version which support ImageStack (1.18)